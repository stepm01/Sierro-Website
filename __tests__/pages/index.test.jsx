import { render } from '@testing-library/react'
import HomePage from '../../src/pages'

test('should match snapshot', function () {
  const { container } = render(<HomePage />)
  expect(container).toMatchSnapshot()
})
