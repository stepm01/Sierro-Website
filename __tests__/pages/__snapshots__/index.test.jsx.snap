// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <input
    class="border-2 border-amber-900 m-5"
    type="text"
    value=""
  />
  <h1
    data-testid="greeting"
  />
</div>
`;
